<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonMultiply">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxNumber2">
       <property name="maximum">
        <double>10000000000000000139372116959414099130712064.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonDivision">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonMinus">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxNumber1">
       <property name="maximum">
        <double>9999999999999999464902769475481793196872414789632.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonAddition">
       <property name="text">
        <string>+</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
